### ISSUE TITLE
Lesson 14: Growth Type

### ISSUE THREAD

**Issue by FinnMM** [GDQuest/learn-gdscript#896]
The growth type in one of the examples is not exponential: it is a multiple (multiplier). Or it is simply confusing and unclear to follow, since the explanation is mixing whether or not it multiplies on top of itself for each assignment or using some base status * multiple to reassign altogether.

**Comment by NathanLovato** [comment]
Is it about this example?

```gdscript
var max_health = 100

func level_up():
    max_health *= 1.1
```

Or is it another one? Could you please maybe share a screenshot of the example in question?

**Comment by FinnMM** [comment]
I think it's just a little confusing to follow, since you talk about exponential stuff and then talk about multiplication. It might have made more sense to explain multiplication first, and then talk about exponentiation.

![image](https://github.com/GDQuest/learn-gdscript/assets/69992917/cab073ee-95f1-40bc-ac8e-81e753b09366)
So we have a damage with a level multiplier following the start of your exponential explanation.

![image](https://github.com/GDQuest/learn-gdscript/assets/69992917/2d3fedf6-f5aa-41d5-bc6d-51ea86a7c496)
And here, again, the example isn't all that clear, I don't think. It felt like you were going to say something about exponential stuff but ended up not doing that.

![image](https://github.com/GDQuest/learn-gdscript/assets/69992917/4ffc2133-ce48-4d0b-bd1d-a17086989c56)
Finally, this image feels like it was supposed to go right after the explanation of exponential stuffs at the start. Though the explanation seems a little unclear about whether or not it is just multiplying by some constant factor, or multiplying + assigning, which would be exponential.

It's not too drastic, but I'm just letting you know that the flow of this passage seems to make it difficult to follow easily, and I'm guessing ease of reading is something you want. It feels like an explanation that gets interrupted by another explanation before finishing the original explanation, which makes for some really broken reading.

**Comment by NathanLovato** [comment]
You're right, thanks for the detailed breakdown! The lesson overall needs a pass to improve the flow and clarity.
